src/en/guide/deployment/caddy.md

```diff
@@ -0,0 +1,74 @@
+# Caddy Deployment
+
+## Introduction
+
+Caddy is a state-of-the-art web server and proxy that supports up to HTTP/3. Its simplicity lies in its minimalistic configuration and the inbuilt ability to automatically procure TLS certificates for your domains from Let's Encrypt. In this setup, we will configure the Sanic application to serve locally at 127.0.0.1:8001, with Caddy playing the role of the public-facing server for the domain example.com.
+
+You may install Caddy from your favorite package menager on Windows, Linux and Mac. The package is named `caddy`.
+
+## Proxied Sanic app
+
+```python
+from sanic import Sanic
+from sanic.response import text
+
+app = Sanic("proxied_example")
+
+@app.get("/")
+def index(request):
+    # This should display external (public) addresses:
+    return text(
+        f"{request.remote_addr} connected to {request.url_for('index')}\n"
+        f"Forwarded: {request.forwarded}\n"
+    )
+```
+
+To run this application, save as `proxied_example.py`, and use the sanic command-line interface as follows:
+
+```bash
+SANIC_PROXIES_COUNT=1 sanic proxied_example --port 8001
+```
+
+Setting the SANIC_PROXIES_COUNT environment variable instructs Sanic to trust the X-Forwarded-* headers sent by Caddy, allowing it to correctly identify the client's IP address and other information.
```

Comment:
Should we add a note that the count might need to be different if there are more layers in front of caddy?

Reply:
I would think that is obvious to anyone in such situation, and also from the name, and it is documented elsewhere.